src/test/java/teammates/test/cases/webapi/UpdateFeedbackResponseActionTest.java

```diff
@@ -66,7 +66,7 @@ protected void prepareTestData() {
     @Test
```

Comment:
Is this still relevant?

Reply:
Do you mean the `@Test` annotation or the file?

Reply:
The `@Test` annotation